Python/PlenoInterview/Problem1/order.py

```diff
@@ -0,0 +1,40 @@
+class order:
+
+    def __init__(self, numPeople):
+        self.orderItems = []
+        self.numPeople = numPeople
+
+    # Functions of class
+    def getOrderTotal(self):
+        total = 0
+        for order in self.orderItems:
+            total += order.getTotalPrice()
+        return total
+
+    def getTotalPerPerson(self):
+        return self.getOrderTotal()/self.numPeople
+
+    def getOrderItemByName(self, dishName):
+        for order in self.orderItems:
+            if (order.dishName == dishName):
+                return order
+        return None
+
+    def addNewOrderItem(self, orderItem):
```

Comment:
It would be better to add a `addOrderItem` function that verifys if the Order exsits and calls either
`addNewOrderItem` or `addExistingOrderItem`.

```
def addOrderItem(self, oderItem):
  if self.getOrderItemByName(orderItem.dishName) is None:
    self.addNewOrderItem(orderItem)
  else:
    self.addExistingOrderItem(orderItem)
```

(ref2)